tests/tools/onert_train/src/args.cc

```diff
@@ -274,6 +274,17 @@ void Args::Parse(const int argc, char **argv)
     if (!vm.count("modelfile") && !vm.count("nnpackage") && !vm.count("path"))
       throw boost::program_options::error(
         std::string("Require one of options modelfile, nnpackage, or path."));
+
+    if (!vm.count("data_length"))
+      throw boost::program_options::error(std::string("data_length option is mandatory."));
+
+    auto is_given = [&vm](const std::string &s) -> bool { return not vm[s].defaulted(); };
+
+    if (is_given("batch_size") && is_given("learning_rate") && is_given("loss") &&
+        is_given("optimizer"))
+    {
+      _is_training_info = true;
+    }
```

Comment:
It would be better to check if there is a given parameter in main function instead of using `_is_training_info` variable.

Reply:
I agree.

But for now, It is hard to check in the main. 
Because `Arg` has a default value, It is hard to figure out whether the value is given or default in main. 

https://github.com/Samsung/ONE/blob/8243c77d683af17786e3ae6e752dd28ffb98bd41/tests/tools/onert_train/src/args.cc#L212-L222